module3/assessments/week3-6490.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Week 3 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_Mod3Week3Assessment)
```

Comment:
I decided to name the repo with the random 4 numbers for that assessment, realized that we have the same thing with easy to find future assessments if we name the repos all the same.